dbi/dbi.go

```diff
@@ -153,7 +150,6 @@ func (dbiPlg *DbiPlugin) getMetrics() (map[string]interface{}, error) {
 	metrics := map[string]interface{}{}
 
 	err := openDBs(dbiPlg.databases)
-	defer closeDBs(dbiPlg.databases)
```

Comment:
What is the reason behind removing `defer` statement and moving `closeDBs()` to the end of function?